hydra-main/src/main/java/com/addthis/hydra/job/spawn/AvailabilityDomain.java

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
feel like this deserves a better name, because AvailabilityDomain is a thing in OCI. Maybe ProrityLevel or SpreadPrioirty or GeoLevel... ?